tests/test_validators.py

```
@@ -107,5 +94,5 @@ def test_region_group_check():
 
 
 @pytest.mark.parametrize('value, expected', params)
-def test_no_slash(value, expected):
-    assert no_slash_or_pipe(model=None, element=value) == expected
+def test_no_slash(value: str | int, expected: bool) -> None:
```

Ruff (0.14.4)

Boolean-typed positional argument in function definition

(FBT001)